pkg/apis/monitoring/v1/probe_types.go

```diff
@@ -30,7 +30,7 @@ const (
 // +k8s:openapi-gen=true
 // +kubebuilder:resource:categories="prometheus-operator",shortName="prb"
 
-// Probe defines monitoring for a set of static targets or ingresses.
+// The `Probe` custom resource definition (CRD) allows to declarative define how groups of ingresses and static targets should be monitored. Besides the target, the `Probe` object requires a `prober` which is the service that monitors the target and provides metrics for Prometheus to scrape. Typically, this is achieved using the [blackbox exporter](https://github.com/prometheus/blackbox_exporter).
```

Comment:
WDYM by ingresses in this context?

Reply:
I moved the description from the design doc so I don't have the exact idea. According to me, maybe it means to define how Prometheus should monitor the endpoints exposed by Ingress.

Reply:
Should I add the link to avoid confusion?